auacm/app/modules/submission_manager/views.py

```diff
@@ -35,7 +35,7 @@ def submit():
 
     attempt = models.Submission(
         username=current_user.username,
-        pid=request.form['pid'].lower(),
+        pid=request.form['pid'],
```

Comment:
So the database uses camel case for the pid's, but our data folders that hold judge input/output are all lowercase, which is probably the source of this bug. When we start using the website to upload new problems (what I'm working on right now), we should probably look to staying consistent by switching the directory names to mixed case.